src/Monolog/Handler/ErrorLogHandler.php

```diff
@@ -71,13 +71,14 @@ protected function getDefaultFormatter(): FormatterInterface
      */
     protected function write(array $record)
     {
-        if ($this->expandNewlines) {
-            $lines = preg_split('{[\r\n]+}', (string) $record['formatted']);
-            foreach ($lines as $line) {
-                error_log($line, $this->messageType);
-            }
-        } else {
+        if (!$this->expandNewlines) {
```

Comment:
I'm still not getting ,why falsy condition is better, when you still needs to handle both cases, truthy and falsy.